ui/tooltip/src/Tooltip.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import { styled, theme } from "@washingtonpost/wpds-theme";
+import { TooltipContent } from "./TooltipContent";
+import * as TooltipPrimitive from "@radix-ui/react-tooltip";
+
+const StyledTrigger = styled(TooltipPrimitive.Trigger, {
+  color: theme.colors.primary,
+});
+
+type TooltipTriggerType = React.ComponentPropsWithRef<typeof StyledTrigger>;
+
+const TooltipTrigger = React.forwardRef<HTMLButtonElement, TooltipTriggerType>(
+  ({ children, ...props }, ref) => (
+    <StyledTrigger {...props} ref={ref} asChild>
+      {children}
+    </StyledTrigger>
+  )
+);
+
+TooltipTrigger.displayName = "TooltipComponentTrigger";
```

Comment:
I think we should follow convention and say this:
```suggestion
TooltipTrigger.displayName = "TooltipTrigger";
```